gmutex/gmutex.go

```diff
@@ -36,23 +37,42 @@ import (
 // (it is allowed for one goroutine to lock a Mutex
 // and then arrange for another goroutine to unlock it),
 // but it is not safe for concurrent use by multiple goroutines.
+//
+// To use an API-compatible alternative to Google Cloud Storage (such as
+// fake-gcs-server or similar), provide the endpoint by setting the
+// environment variable STORAGE_EMULATOR_HOST prior to creating the
+// Mutex.
 type Mutex struct {
 	_          noCopy
 	bucket     string
 	object     string
 	generation string
 	ttl        int64
+	baseUrl    *url.URL
 }
 
 // New creates a new Mutex at the given bucket and object,
-// with the given time-to-live.
+// with the given time-to-live. To use a fake Google Cloud Storage
+// during tests, provide the API-compatible endpoint hostname
+// in the environment variable STORAGE_EMULATOR_HOST prior to calling
+// this function.
```

Comment:
Would you like me to remove additions to the godocs?

Reply:
Tweaked in a follow up.
Thanks though!